Pod/Classes/BottomSheetComponent/BottomSheetViewController.swift

```diff
@@ -0,0 +1,1521 @@
+//
+//  BottomSheetViewController.swift
+//  Emission
+//
+//  Code reused from https://github.com/52inc/Pulley
```

Comment:
Is it possible to use the Pulley library as-is from their repo? It has a podfile. If not, we should document why somewhere.

Reply:
I looked into it and it didn't work out for a few reasons
1. I wasn't able to import the swift interfaces from Objective-C using `#import <Pulley/Pulley-Swift.h>`
2. There's some tweaks we will need to do for our use case that will be less straightforward to it with Pulley